### ISSUE TITLE
Bug Report: Pause & Resume

### ISSUE THREAD

**Issue by rcoletti116** [bloomkd46/homebridge-iRobot#64]
### Describe The Bug:

When you have the Off Action set to Pause, the device fails to restart on the next command. This is because the Paused state requires a "resume" command and not a "clean" command. 

For reference:

https://github.com/karlvr/homebridge-roomba2/issues/62#issuecomment-923243900
https://github.com/karlvr/homebridge-roomba2/blob/main/src/accessory.ts#L435

### To Reproduce:

1. Set "Off Action" to Pause. 
2. Turn on Roomba device
3. Turn off device -> device will pause
4. Turn on device -> device will error.

### Expected behavior:

The plugin should send a resume command to a paused robot. 

### Logs:

```shell
[25/01/2022, 09:44:18] [iRobot] Set Rosie To pause 
[25/01/2022, 09:44:22] [iRobot] Set Rosie To Clean
```

### Config:

```shell
"offAction": "pause:none",
```

### Node Version:

16.13.2

### NPM Version:

8.3.1

### Homebridge Version:

1.4.0

### Plugin Version:

homebridge-irobot v2.1.14-beta.4

### Operating System:

Raspberian

**Comment by bloomkd46** [comment]
I will fix this in my next release

**Comment by bloomkd46** [comment]
This should be fixed in version 2.1.16
